addons/hr_holidays/tests/test_leave_requests.py

```diff
@@ -7,7 +7,7 @@
 from pytz import timezone, UTC
 
 from odoo import fields
-from odoo.exceptions import ValidationError
+from odoo.exceptions import ValidationError, UserError
```

Comment:
UserError is reimported line 15, you can remove line 15 to pass style checks :)
Detail, but we usually put imports in alphabetical order 
```suggestion
from odoo.exceptions import UserError, ValidationError
```